plugins/pci-mei/fu-pci-mei-plugin.c

```diff
@@ -546,8 +546,10 @@ fu_pci_mei_plugin_add_security_attrs(FuPlugin *plugin, FuSecurityAttrs *attrs)
 	attr = fu_security_attrs_get_by_appstream_id(attrs,
 						     FWUPD_SECURITY_ATTR_ID_SUPPORTED_CPU,
 						     NULL);
-	if (attr != NULL)
+	if (attr != NULL) {
 		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS);
+		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_VALID);
+	}
```

Comment:
can we set fwupd_security_attr_set_result_success() when we create the attr?

Reply:
Hum; we do..
https://github.com/fwupd/fwupd/blob/5fd26aa4c1cabc72f19d689e26e465270abca4f5/src/fu-engine.c#L6445

Reply:
I'm guessing this helps:

@@ -6441,7 +6441,6 @@ fu_engine_ensure_security_attrs_supported_cpu(FuEngine *self)
 
        fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONTACT_OEM);
        fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_MISSING_DATA);
-       fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_VALID);
        fwupd_security_attr_set_result_success(attr, FWUPD_SECURITY_ATTR_RESULT_VALID);
        fu_security_attrs_append(self->host_security_attrs, attr);
 }

Reply:
> I'm guessing this helps:

Yes that works.

Side effect of this is on unsupported (instead of Invalid):
```
✘ Supported CPU:                 Unknown
```

> or we could do:

Yes, this works and correctly says:
```
✘ Supported CPU:                 Invalid
```

Feel free to discard my PR and send that instead.